src/pages/CreateUserPage.jsx

```diff
@@ -402,6 +404,8 @@ function CreateUserPage(props) {
                                         className="form-control rounded-4 bg-light-pastel-blue color-grey fw-medium fs-5 border-0 mb-3"
                                         autoComplete="off"
                                         onChange={(e) => setNewUser({ ...newUser, username: e.target.value })}
+                                        minLength="3"
+                                        required
```

Comment:
```suggestion
                                        minLength="3"
                                        maxLength="20"
                                        required
```
- Username também possui limite superior na API;